src/EventEdition/InfoPanel.vala

```diff
@@ -264,6 +285,31 @@ public class Maya.View.EventEdition.InfoPanel : Gtk.Grid {
                 Gtk.TextBuffer buffer = new Gtk.TextBuffer (null);
                 buffer.text = property.get_comment ();
                 comment_textview.set_buffer (buffer);
+
+                // Identify links in the description and apply a tag to style them
+                var link_regex = new GLib.Regex ("https?://[\\w.-]*(\\/[\\w-\\d]*)*");
```

Comment:
This doesn't account for some types of URLs, for example ones that include `?` parameters. This is common in things like Zoom links. 

[Here's a blog post](https://mathiasbynens.be/demo/url-regex) that goes over some example patterns. I'm not sure if there's any licensing issues involved with copying these, however.